libraries/AP_GPS/AP_GPS_NMEA.cpp

```diff
@@ -361,7 +429,21 @@ bool AP_GPS_NMEA::_term_complete()
         case _GPS_SENTENCE_GPRMC + 2: // validity (RMC)
             _gps_data_good = _term[0] == 'A';
             break;
+        case _GPS_SENTENCE_GPRMC + 12: // Mode indicator (RMC)
+            /* A = Autonomous mode
+             * D = Differential mode
+             * F = Float RTK  (specific for NV08C (not official NMEA 0813))
+             * R = Real Time Kinematic (specific for NV08C (not official NMEA 0813))
+             * E = Estimated (dead reckoning) mode
+             * M = Manual input mode
+             * S = Simulator mode
+             * N = Data not valid
+             */
```

Comment:
What GPS gave you this table? It doesn't match the documentation I can find for the GPRMC message anywhere. According to everything I've found the only standard is A=Data Valid or V=Data not valid (which you don't have listed here)